packages/react-divider/src/components/Divider/Divider.test.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import { Divider } from './Divider';
+import * as renderer from 'react-test-renderer';
+import { ReactWrapper } from 'enzyme';
+import { isConformant } from '../../common/isConformant';
+
+describe('Divider', () => {
+  isConformant({
+    Component: Divider,
+    displayName: 'Divider',
+  });
+
+  let wrapper: ReactWrapper | undefined;
+
+  afterEach(() => {
+    if (wrapper) {
+      wrapper.unmount();
+      wrapper = undefined;
+    }
+  });
+
+  /**
+   * Note: see more visual regression tests for Divider in /apps/vr-tests.
+   */
+  it('renders a default divider', () => {
+    const component = renderer.create(<Divider />);
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
Are the snapshots in the PR ?

Reply:
Im not sure what else is needed, I spoke with Mak and basically just followed what I could determine from Avatar / existing test. Can we discuss what updates / etc is needed for this?

Reply:
Update